dask/bytes/core.py

```diff
@@ -34,7 +35,7 @@ def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
 
     Parameters
     ----------
-    path: string
+    urlpath: string
```

Comment:
I think that many people today associate `url` with a web resource.  If we use this term we should probably include a description below saying the possible options.